### PR TITLE
Add some typos seen in the wild.

### PR DESCRIPTION
**Pull request by jtroup**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22925,6 +22925,7 @@ produkt->product
 produse->produce
 prodused->produced
 produses->produces
+proedure->procedure
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22928]: More variants:
```suggestion
proedural->procedural
proedure->procedure
proedures->procedures
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22945,6 +22946,7 @@ proffesion->profession
 proffesional->professional
 proffesor->professor
 proffessor->professor
+profie->profile
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22949]: Again:
```suggestion
profie->profile
profied->profiled
profier->profiler
profies->profiles
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -31541,6 +31543,7 @@ usinng->using
 usng->using
 usnig->using
 uspported->supported, unsupported,
+usptart->upstart
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:31546]:
```suggestion
usptart->upstart
usptarts->upstarts
```

**Review by peternewman — CHANGES_REQUESTED** [review]
Some more variants

**Comment by luzpaz** [comment]
@jtroup please follow-up

**Pull request merged**